bigtable/metrics_monitoring_exporter.go

```diff
@@ -236,15 +248,15 @@ func (me *monitoringExporter) recordToTimeSeriesPb(m otelmetricdata.Metrics) ([]
 	default:
 		errs = append(errs, errUnexpectedAggregationKind{kind: reflect.TypeOf(m.Data).String()})
 	}
-	return tss, errors.Join(errs...)
+	return tss, me.logErrorAndReturn("recordToTimeSeriesPb", errors.Join(errs...))
 }
 
-func sumToTimeSeries[N int64 | float64](point otelmetricdata.DataPoint[N], metrics otelmetricdata.Metrics, mr *monitoredrespb.MonitoredResource) (*monitoringpb.TimeSeries, error) {
-	interval, err := toNonemptyTimeIntervalpb(point.StartTime, point.Time)
+func (me *monitoringExporter) sumToTimeSeries(point otelmetricdata.DataPoint[int64], metrics otelmetricdata.Metrics, mr *monitoredrespb.MonitoredResource) (*monitoringpb.TimeSeries, error) {
```

Comment:
Need access to error logger. So, changing this function to a method on monitoringExporter.
Also, removing generics since we only use int64 and Go does not allow generic methods.

Below code would throw error `method must have no type parameters`
```
func (me *monitoringExporter) sumToTimeSeries[N int64 | float64](point otelmetricdata.DataPoint[int64], metrics otelmetricdata.Metrics, mr *monitoredrespb.MonitoredResource)
```